ui/src/main.js

```diff
@@ -1,10 +1,27 @@
 import Vue from "vue";
+import VueRouter from 'vue-router'
 import App from "./App.vue";
 
 import "./assets/app.css";
 
 Vue.config.productionTip = false;
 
+Vue.use(VueRouter)
+
+const Foo = { template: '<div></div>' }
+
+const router = new VueRouter({
+  mode: 'history', // use hashless route
+  routes: [
+    { path: '/getting-started', name: "getting-started", component: Foo},
+    { path: '/tools', name: "tools", component: Foo},
+    { path: '/extensions', name: "extensions", component: Foo},
+    { path: '/', redirect: '/getting-started'},
+    { path: "*", redirect: '/getting-started'}
+  ]
+})
```

Comment:
```suggestion
const BlankTemplate = { template: '<div></div>' }

const router = new VueRouter({
  mode: 'history', // use hashless route
  routes: [
    { path: '/getting-started', name: "getting-started", component: BlankTemplate },
    { path: '/tools', name: "tools", component: BlankTemplate },
    { path: '/extensions', name: "extensions", component: BlankTemplate },
    { path: '/', redirect: '/getting-started'},
    { path: "*", redirect: '/getting-started'}
  ]
})
```
Can we use something a bit more descriptive as a class name?